### PR TITLE
Add support for @scoped packages

### PR DESCRIPTION
**Pull request by duailibe**
Hey, thanks for this plugin!

This adds support for newly added support for @scoped packages in ESLint v5

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,9 +1,18 @@
 'use strict';
 
 const plugin = require('../..');
-const pluginName = require('../../package.json').name.replace(/^eslint-plugin-/, '');
 const selfPlugin = Object.assign({}, plugin);
 
+const pkgName = require('../../package.json').name;
+let pluginName;
+if (pkgName[0] === "@") {
+  const parts = pkgName.split('/');
```
Comment (not-an-aardvark) [inline index.js]: Should this `parts` variable be removed? It doesn't seem to be used anywhere.

**Review by not-an-aardvark — CHANGES_REQUESTED** [review]
Thanks for the PR, I left a minor comment.

File: index.js (hunk 1 above)
Comment (duailibe) [reply index.js]: Yeah I've changed it to use a regex and forgot to remove the split

Sorry about that

**Comment by duailibe** [comment]
@not-an-aardvark Sorry to bother, do you think you can take a look at this soon and cut a release? Thanks!

**Review by not-an-aardvark — APPROVED** [review]

**Comment by not-an-aardvark** [comment]
Published as v1.1.0.

**Pull request merged**